src/pages/settings/RedditDataMigratePage.tsx

```diff
@@ -108,7 +146,19 @@ export default function RedditMigratePage() {
       <IonHeader>
         <IonToolbar>
           <IonButtons slot="start">
-            <IonBackButton defaultHref="/settings" text="Settings" />
+            {subs ? (
+              <IonButton
+                onClick={(e) => {
```

Comment:
This should probably navigate to a separate page if we're going to allow navigating back.

The migrate page is quite terse/simple, so this will require a bit of refactor if we want to increase the complexity with saving migrate links.